lib/taskana-core/src/test/java/acceptance/task/QueryTasksAccTest.java

```diff
@@ -41,29 +42,193 @@
+    List<TaskSummary> tasks = taskService.createTaskQuery().ownerLongNameIn(longName).list();
+
+    assertThat(tasks)
+        .hasSize(23)
+        .extracting(TaskSummary::getOwnerLongName)
+        .doesNotContainNull()
+        .containsOnly(longName);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_SetOwnerLongNameOfTask_When_FilteringWithOwnerLongNameNotIn() {
+    taskanaEngineConfiguration.setLongNameIncludedInQuery(false);
+    List<TaskSummary> tasks =
+        taskService
+            .createTaskQuery()
+            .idIn(
+                "TKI:000000000000000000000000000000000000",
+                "TKI:000000000000000000000000000000000027")
+            .ownerLongNameNotIn("Eifrig, Elena - (user-1-2)")
+            .list();
+
+    assertThat(tasks).hasSize(1);
+    assertThat(tasks.get(0))
+        .extracting(TaskSummary::getOwnerLongName)
+        .isEqualTo("Mustermann, Max - (user-1-1)");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_SetOwnerLongNameOfTask_When_FilteringWithOwnerLongNameLike() {
+    taskanaEngineConfiguration.setLongNameIncludedInQuery(false);
+    List<TaskSummary> tasks = taskService.createTaskQuery().ownerLongNameLike("%1-2%").list();
+
+    assertThat(tasks)
+        .hasSize(23)
+        .extracting(TaskSummary::getOwnerLongName)
+        .doesNotContainNull()
+        .containsOnly("Eifrig, Elena - (user-1-2)");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_SetOwnerLongNameOfTask_When_FilteringWithOwnerLongNameNotLike() throws Exception {
+    taskanaEngineConfiguration.setLongNameIncludedInQuery(false);
+    List<TaskSummary> tasks = taskService.createTaskQuery().ownerLongNameNotLike("%1-1%").list();
+
+    assertThat(tasks)
+        .hasSize(23)
+        .extracting(TaskSummary::getOwnerLongName)
+        .doesNotContainNull()
+        .containsOnly("Eifrig, Elena - (user-1-2)");
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_SetOwnerLongNameOfTaskToNull_When_OwnerNotExistingAsUserInDatabase() {
+    taskanaEngineConfiguration.setLongNameIncludedInQuery(true);
+    List<TaskSummary> tasks =
+        taskService.createTaskQuery().idIn("TKI:000000000000000000000000000000000041").list();
+
+    assertThat(tasks).hasSize(1);
+    ThrowingCallable call =
+        () -> taskanaEngine.getUserService().getUser(tasks.get(0).getOwner()).getLongName();
+    assertThatThrownBy(call).isInstanceOf(UserNotFoundException.class);
+    assertThat(tasks.get(0)).extracting(TaskSummary::getOwnerLongName).isNull();
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_OrderByOwnerLongName_When_QueryingTask() {
+    taskanaEngineConfiguration.setLongNameIncludedInQuery(false);
+    List<TaskSummary> tasks = taskService.createTaskQuery().orderByOwnerLongName(ASCENDING).list();
+    assertThat(tasks)
+        .hasSize(88)
+        .extracting(TaskSummary::getOwnerLongName)
+        .extracting(Objects::nonNull)
```

Comment:
i don't think this is what you wanted to do....you're extracting if the ownerLong name is null or not...which returns true or false....and then the following .isSorted will fail....if you wanted to filter all null ownerLongnames you would need to do that differently